src/nvim/testdir/test_breakindent.in

```diff
@@ -101,7 +101,7 @@ fygjyl:let line2 = @0
 :$put =line2
 :"
 :let g:test="Test 14: breakindent + visual blockwise delete #1"
-:set all& breakindent
+:set all& breakindent viminfo+=nviminfo
```

Comment:
I would suggest to use `shada+=nX-test-breakindent.shada`. `&viminfo` is now a deprecated option, ShaDa uses `.shada` extension and it is preferred to have at least test-file-local names of the temporary files.